### PR TITLE
Downloader UI revisions

### PR DESCRIPTION
**Pull request by bsechelski-pointblue**
- Select all species when guild is selected
 - Add instructions
 - Add titles for controls and UI descriptions
 - Remove default mockup username
 - Remove include special projects checkbox

### CODE REVIEW

File: resources/views/partials/filters.blade.php [hunk 1]
```diff
@@ -1,35 +1,30 @@
-<h1>Download Data</h1>
-<p>
+<h1>Welcome to AKN Data Downloader</h1>
+<p>AKN Data Downloader is an application for downloading available field observations provided by partners of the Avian Knowledge Network, hosted by <a href="http://www.pointblue.org/our-science-and-services/conservation-tools/data-solutions/" target="_blank"> Point Blue Data Solutions</a>.</p>
+<p>Please follow the following steps to review and download observation data:</p>
```
Comment (stereocilia) [inline resources/views/partials/filters.blade.php]: Please use the following steps...

File: resources/views/partials/filters.blade.php [hunk 2]
```diff
@@ -1,35 +1,30 @@
...
 <ol>
-    <li>Use the options below to filter the data want to download</li>
-    <li>Click the 'Search' button to see a summary of the data matching your selections</li>
-    <li>Click the 'Download' button at the bottom of the Search Summary to download your data</li>
+    <li>Select the tab for the data observation type you want to use.</li>
```
Comment (stereocilia) [inline resources/views/partials/filters.blade.php]: There's only one tab right now, so we can omit this

File: resources/views/partials/filters.blade.php [hunk 3]
```diff
@@ -1,35 +1,30 @@
...
+    <li>Select from the Filters below to choose the subset of data you want to download.</li>
+    <li>Choose the 'Search' button to see a summary of the data matching your selections.</li>
```
Comment (stereocilia) [inline resources/views/partials/filters.blade.php]: I understand not wanting to use 'Click', but 'Choose' is strange. Can we just say 'Use'?

File: resources/views/partials/filters/species.blade.php [hunk 4]
```diff
@@ -1,4 +1,5 @@
 <h3>Species</h3>
+<p>Choose species to refine your results, or select "All Species" to retrieve all observed species from the selected project. Only those species that have been observed shown in this list.  Guilds are provided from existing projects as a convenient way to shorten the species list.</p>
```
Comment (stereocilia) [inline resources/views/partials/filters/species.blade.php]: Only those species that have been observed ARE shown in this list?

File: resources/views/partials/summary.blade.php [hunk 5]
```diff
@@ -12,6 +12,7 @@
         <div class="row">
             <div class="col-md-12">
                 <h2>Search Summary</h2>
+                <p> The following table summarized how many observations match your selection of filters. Select the Download button to get a copy of the data you have selected.  Please note that the system is limited at this time to downloading no more than 10,000 records at once.</p>
```
Comment (stereocilia) [inline resources/views/partials/summary.blade.php]: The following table summarizes how many observations match your filters choices. Use the Download button...

File: resources/angular/view/Filters/FiltersController.js [hunk 6]
```diff
@@ -183,6 +185,15 @@
             ;
         }
 
+        function birdCodeForSpecies(element){
+            return element.birdCode;
+        }
+
+        function setSpeciesSelected(species) {
+            vm.selectedFilter.species.all =  false;
```
Comment (stereocilia) [inline resources/angular/view/Filters/FiltersController.js]: is this necessary? i don't believe the `handleGuildsChanged` event can fire if `vm.selectedFilter.species.all == true`

Comment (bsechelski-pointblue) [inline resources/angular/view/Filters/FiltersController.js]: may not be I'll try testing without

**Pull request merged**